RichEditorViewSample/RichEditorViewSample/KeyboardManager.swift

```diff
@@ -36,8 +36,8 @@ class KeyboardManager: NSObject {
         Starts monitoring for keyboard notifications in order to show/hide the toolbar
     */
     func beginMonitoring() {
-        NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(KeyboardManager.keyboardWillShowOrHide(_:)), name: UIKeyboardWillShowNotification, object: nil)
-        NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(KeyboardManager.keyboardWillShowOrHide(_:)), name: UIKeyboardWillHideNotification, object: nil)
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: Selector("KeyboardManager.keyboardWillShowOrHide:"), name: UIKeyboardWillShowNotification, object: nil)
```

Comment:
Yea, I simply switched it back since 7.3 wasn't out and switching it back every time i updated a branch was getting annoying